pkg/loop/internal/pb/relayerset/relayerset.proto

```diff
@@ -128,6 +125,19 @@ service RelayerSet {
   rpc ContractReaderUnbind(ContractReaderUnbindRequest) returns (google.protobuf.Empty) {}
   rpc ContractReaderStart(ContractReaderStartRequest) returns (google.protobuf.Empty) {}
   rpc ContractReaderClose(ContractReaderCloseRequest) returns (google.protobuf.Empty) {}
+
+  rpc EVMGetTransactionFee(EVMGetTransactionFeeRequest) returns (loop.chain_capabilities.evm.GetTransactionFeeReply) {}
+  rpc EVMCallContract(EVMCallContractRequest) returns (loop.chain_capabilities.evm.CallContractReply);
+  rpc EVMFilterLogs(EVMFilterLogsRequest) returns (loop.chain_capabilities.evm.FilterLogsReply);
+  rpc EVMBalanceAt(EVMBalanceAtRequest) returns (loop.chain_capabilities.evm.BalanceAtReply);
+  rpc EVMEstimateGas(EVMEstimateGasRequest) returns (loop.chain_capabilities.evm.EstimateGasReply);
+  rpc EVMTransactionByHash(EVMTransactionByHashRequest) returns (loop.chain_capabilities.evm.TransactionByHashReply);
+  rpc EVMTransactionReceipt(EVMReceiptRequest) returns (loop.chain_capabilities.evm.TransactionReceiptReply);
+  rpc EVMLatestAndFinalizedHead(LatestHeadRequest) returns (loop.chain_capabilities.evm.LatestAndFinalizedHeadReply);
+  rpc EVMQueryTrackedLogs(EVMQueryTrackedLogsRequest) returns (loop.chain_capabilities.evm.QueryTrackedLogsReply);
+  rpc EVMRegisterLogTracking(EVMRegisterLogTrackingRequest) returns (google.protobuf.Empty);
+  rpc EVMUnregisterLogTracking(EVMUnregisterLogTrackingRequest) returns (google.protobuf.Empty);
+  rpc EVMGetTransactionStatus(EVMGetTransactionStatusRequest) returns (GetTransactionStatusReply);
```

Comment:
Why don't we define these as a separate `EVMServiceSet` service, so we can drop all the `EVM` prefixes